test/e2e/framework/provider.go

```diff
@@ -43,13 +43,16 @@ func RegisterProvider(name string, factory Factory) {
 }
 
 func init() {
-	// "local" or "skeleton" can always be used.
+	// register some known null providers
 	RegisterProvider("local", func() (ProviderInterface, error) {
 		return NullProvider{}, nil
 	})
 	RegisterProvider("skeleton", func() (ProviderInterface, error) {
 		return NullProvider{}, nil
 	})
+	RegisterProvider("kubernetes-anywhere", func() (ProviderInterface, error) {
```

Comment:
why?  

skeleton or local should be sufficient for our automation.

Reply:
no, `--provider=kubernetes-anywhere` is still needed.
the alternative is to modify all the jobs to apps `skeleton` instead?